apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4CoverageResourceProvider.java

```diff
@@ -138,6 +138,9 @@ public Coverage read(@IdParam IdType coverageId) {
           new IdDt(Beneficiary.class.getSimpleName(), String.valueOf(beneficiaryId)));
     }
 
+    // Add bene_id to MDC logs
+    TransformerUtilsV2.logBeneIdToMdc(beneficiaryId);
+
     Coverage coverage =
         CoverageTransformerV2.transform(metricRegistry, coverageIdSegment.get(), beneficiaryEntity);
     return coverage;
```

Comment:
I took a quick look and I think we are missing calls in EOB and Patient as well

Reply:
I looked in both EOB and Patient and didnt find anything. Maybe those were methods that @read and @search call?

Reply:
All methods have now been addressed. Will modify where to possibly log bene_id if there are issues with the tests/suggestions from reviewers.